backend/btrixcloud/crawls.py

```diff
@@ -713,6 +717,17 @@ async def filter_crawl_queue(self, crawl_id, regex):
 
         return num_removed
 
+    async def get_errors_from_redis(self, crawl_id):
+        """Get crawl errors from Redis and optionally store in mongodb."""
+        try:
+            redis = await self.get_redis(crawl_id)
+            errors = await redis.lrange(f"{crawl_id}:e", 0, -1)
```

Comment:
Probably fine for now, but may want to do this incrementally, depending on how many errors we'll have or limit error size, eg. if this is pulling down, say a 10000+ list, this may likely crash the redis pod, given current memory limit

We could fetch list in batches, or impose a max size limit, as discussed previously.
Maybe simplest thing to do here add an offset, and default size, so frontend can handle pagination, eg. `errors?offset=...`